infra/gcp/README.md

```diff
@@ -262,15 +262,15 @@ Using the Google cloud console, create a VM in Compute Engine:
   - Region / zone: use the same as the GKE cluster
   - n1-standard-8
   - OS and storage
-    - Ubuntu 22.04 LTS
+    - Ubuntu 24.04 LTS
```

Comment:
I think this is actually only used as a helper VM to run some jobs up in the cloud. But using 24 instead of 22 is a good change anyway so 👍